### DISCUSSION TITLE
#67 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by yanyanho** [tog-ETH-er/togETHer#52] [General]
Agenda:

Community Updates
Fireside Chat
会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/kcf-vhtb-rwx

**Comment by yanyanho** [comment]
# Dapp-Learning

## repo update

7个PR #970 ～ #976

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了场分享会
 
- 《合约钱包实例分析》by Jianqi
   - Youtube: https://www.youtube.com/watch?v=yhwW493Eo48&t=5s
   - Bilibili: https://www.bilibili.com/video/BV1Gk4y1P7LR/?spm_id_from=333.999.0.0

- 《密码学多签系列8 zkSnark--Groth16证明系统》by Lynndell
   - Youtube:
     https://www.youtube.com/watch?v=bK-rXiCqvto 
  - Bilibili: 
     https://www.bilibili.com/video/BV1Jh4y177QS/?spm_id_from=333.999.0.0

**Comment by 0xPabloLI** [comment]
## PlanckerDAO

- Web3英语共学，+706线上版，+四海社区线下版
- AAstar whitelist
- [Remix Chinese Community](https://github.com/PlanckerLabs/remix-china/blob/main/weekly-report/7.10-7.16.md)

**Comment by Riely007** [comment]
**以太坊翻译小组**
- 日常 review 进行中
